configs/retinanet/retinanet_r18_fpn_1x_coco.py

```diff
@@ -0,0 +1,15 @@
+_base_ = [
+    '../_base_/models/retinanet_r50_fpn.py',
+    '../_base_/datasets/coco_detection.py',
+    '../_base_/schedules/schedule_1x.py', '../_base_/default_runtime.py'
+]
+# data
+data = dict(samples_per_gpu=8)
```

Comment:
According to the principle of linear scaling, the LR here should be 0.005, do you have a comparison performance?

Reply:
@hhaAndroid 
I compare it with another mode with lr 0.001, not 0.005, and got 31.4 AP.

Reply:
OK